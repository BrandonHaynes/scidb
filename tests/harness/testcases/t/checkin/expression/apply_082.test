--setup
create array expr1 <a:int64, b:int64>[x=1:5,5,0, y=1:5,5,0]
load(expr1,'${TEST_DATA_DIR}/expr1_1.txt')

--test
apply(expr1, a2, 5 / 5 - 3 + 2 + 4 - 3 - 4 * 2 / (4 + 4 * b + b / (4 + 4 * b / 5 + 1 - 4) - 2 / a + 4 + 2 - a) / 2 - a)

--cleanup
remove (expr1)
