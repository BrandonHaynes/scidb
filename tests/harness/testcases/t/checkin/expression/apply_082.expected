Query was executed successfully

{x,y} a,b
{1,1} 2,4
{1,2} 6,8
{1,3} 10,12
{1,4} 14,16
{1,5} 18,20
{2,1} 22,24
{2,2} 26,28
{2,3} 30,32
{2,4} 34,36
{2,5} 38,40
{3,1} 42,44
{3,2} 46,48
{3,3} 50,52
{3,4} 54,56
{3,5} 58,60
{4,1} 62,64
{4,2} 66,68
{4,3} 70,72
{4,4} 74,76
{4,5} 78,80
{5,1} 82,84
{5,2} 86,88
{5,3} 90,92
{5,4} 94,96
{5,5} 98,100

{x,y} a,b,a2
{1,1} 2,4,-1
{1,2} 6,8,-5
{1,3} 10,12,-9
{1,4} 14,16,-13
{1,5} 18,20,-17
{2,1} 22,24,-21
{2,2} 26,28,-25
{2,3} 30,32,-29
{2,4} 34,36,-33
{2,5} 38,40,-37
{3,1} 42,44,-41
{3,2} 46,48,-45
{3,3} 50,52,-49
{3,4} 54,56,-53
{3,5} 58,60,-57
{4,1} 62,64,-61
{4,2} 66,68,-65
{4,3} 70,72,-69
{4,4} 74,76,-73
{4,5} 78,80,-77
{5,1} 82,84,-81
{5,2} 86,88,-85
{5,3} 90,92,-89
{5,4} 94,96,-93
{5,5} 98,100,-97

Query was executed successfully

